src/runtime/vm/executable.cc

```diff
@@ -273,6 +273,13 @@ void Executable::SavePrimitiveOpNames(dmlc::Stream* strm) {
     primitive_names[packed_index] = it.first;
   }
   strm->Write(primitive_names);
+  // TODO(tkonolige): cannot serialize ObjectRefs with dmlc's serializer.
```

Comment:
If this is not needed, maybe can be removed?

Reply:
It is needed, saving and loading an executable will not preserve the extra op attributes.